compiler/modules/print.llvm/llvm.bal

```diff
@@ -69,11 +69,68 @@ public function constInt(IntType ty, int val) returns ConstValue {
     return new ConstValue(ty, val.toString());
 }
 
+final string pINF = (1.0/0.0).toBitsInt().toHexString().toUpperAscii();
```

Comment:
Include the "0x" in these.

pINF is not camel case. I suggest posInf or positiveInfinity.

Reply:
Fixed